### PR TITLE
Move rule writing articles to the top level

### PR DESCRIPTION
**Pull request by adamkvitek**
# Thanks for improving Semgrep Docs 😀

### Please ensure

- [ ] A subject matter expert (SME) reviews the content
- [x] A technical writer reviews the content or PR
- [x] This change has no security implications or else you have pinged the security team
- [ ] Redirects are added if the PR changes page URLs

### CODE REVIEW

**Comment by adamkvitek** [comment]
TODO: check links to these articles - they all need to be fixed potentially.

File: sidebars.js
Comment (s-santillan) [inline sidebars.js:164]: I'd move playground and trophy-case to the bottom of the list. Just a suggestion though.

**Comment by s-santillan** [comment]
@adamkvitek How about we change the header to "Learn Semgrep syntax" so it's clear that the users are learning about the language. (Semgrep isn't a full programming language, so the correct term is syntax, fyi.). Can you also try taking a screenshot so we can see the effect?

**Review by s-santillan — COMMENTED** [review]
We didn't move any physical files, so the links inside the docs should just work, right? I'll approve once you can confirm. Thanks @adamkvitek

**Comment by pabloest** [comment]
@s-santillan I agree about the header referring to something that we want to achieve.

File: sidebars.js
Comment (adamkvitek) [reply sidebars.js:165]: Maybe Playground first and Trophy-case as the last? @s-santillan

Comment (adamkvitek) [reply sidebars.js:165]: Ah... but there needs to be an explanation about writing rules first before we introduce Playground...

**Comment by adamkvitek** [comment]
@pabloest @s-santillan what do you think about this layout now?
Screenshot 2022-09-08 at 16 48 31

- I still feel like... it is hard to fit the trophy-case anywhere, but it may be fine where it is now in this suggestion.
- Should I rewire the filesystem also? It will take some work, but maybe it will be nicer for maintainers (consistent folders in `docs` directory synced with the left sidebar as we will agree upon it here. I may need to check the `semgrep` repository also for any links which may get broken, here also of course.

**Review by s-santillan — APPROVED** [review]
lgtm

**Pull request merged**
